handlers/guest_links.go

```diff
@@ -63,6 +63,40 @@ func (s Server) guestLinksDelete() http.HandlerFunc {
 	}
 }
 
+func (s *Server) guestLinksDisable() http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		id, err := parseGuestLinkID(mux.Vars(r)["id"])
+		if err != nil {
+			log.Printf("failed to parse guest link ID %s: %v", mux.Vars(r)["id"], err)
+			http.Error(w, fmt.Sprintf("Invalid guest link ID: %v", err), http.StatusBadRequest)
+			return
+		}
+
+		if err := s.getDB(r).DisableGuestLink(id); err != nil {
+			log.Printf("failed to delete guest link: %v", err)
+			http.Error(w, fmt.Sprintf("Failed to delete guest link: %v", err), http.StatusInternalServerError)
+			return
+		}
```

Comment:
I haven't been good about doing this elsewhere, but if a handler never returns any content on success, it should end with:

```go
w.WriteHeader(http.StatusNoContent)
```

And we should check for the correct status code in unit tests.